src/selection-tools.js

```diff
@@ -249,24 +249,50 @@ html, body {
     width: 100%;
 }
 body {
+    flex: 1;
     display: flex;
     flex-direction: column;
 }
-select {
+header {
+    flex: 1;
+    padding: 8px;
+    display: none;
+}
+header button {
     width: 100%;
+    margin-top: 6px;
+}
+header button:first-child {
+    margin-top: 0;
 }
 main {
     flex: 1;
+    display: flex;
     overflow-y: auto;
     margin: 8px 0;
 }
+footer {
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    width: 100%;
+}
+footer div {
+    white-space: nowrap;
```

Comment:
The text will not always fit. You might try setting the flex-basis to max-content or fit-content but still allow it to shrink and wrap. The same applies to the <a>. Also should probably add a gap so that the texts won't run together.

Reply:
There are no such long names. I added a fix for correct output.

![Screenshot from 2024-09-24 07-54-05](https://github.com/user-attachments/assets/3c4436d7-c3e7-41a3-863a-7c39f92547fb)

Reply:
It's very much possible that the line would not fit. The texts are localized and in other languages they could be much longer than they are in English. Also the font is set to `menu`, so the text could also become too long depending on the font set by the user.

Reply:
No matter how long the text is, the layout will not float, and the user will find what he needs in the pop-up. I fixed it.

Reply:
+ fix big "by Google"